docs/storybook/.storybook/preview.js

```diff
@@ -36,31 +38,19 @@ export const globalTypes = {
 export const decorators = [
   (Story, context) => {
     return (
-      <div class="theme-wrap">
+      <div className={context.globals.theme === 'all' ? 'theme-wrap-grid' : 'theme-wrap'}>
```

Comment:
Did I miss it, or is there no `theme-wrap` class? Is this on purpose?

Reply:
Great catch!